.env.template

```
@@ -235,6 +235,7 @@ RATE_LIMIT=16
 MESSAGES_BEFORE_LOGIN=5# how many messages a user can send in a conversation before having to login. set to 0 to force login right away
 
 PUBLIC_GOOGLE_ANALYTICS_ID=G-8Q63TH4CSL
+PUBLIC_PLAUSIBLE_SCRIPT_URL="/js/script.js"
```

```suggestion
PUBLIC_PLAUSIBLE_SCRIPT_URL="https://huggingface.co/js/script.js"
```

Maybe?

Because otherwise running inside the space (https://huggingface.co/spaces/huggingchat/chat-ui) it won't work (maybe not important)

---

I think most usage goes through the proxied version but doesn't hurt to have it :)

---

well inside the standard Space URL it's already tracked at the hub page level anyways

So we'll have twice the same script (inside and outside the iframe)